templates/credentials/your-badges.html

```diff
@@ -22,46 +22,54 @@ <h1>Your badges</h1>
             </div>
         </div>
         <div class="row">
-            <table aria-label="Badges earned">
-                <thead>
-                    <tr>
-                        <th>Badge</th>
-                        <th>Exam</th>
-                        <th>Date of Issue</th>
-                        <th>Actions</th>
-                    </tr>
-                </thead>
-                <tbody>
-
-                    {% for badge in badges %}
+            {% if badges['message'] %}
+                <div class="p-notification--information">
+                    <div class="p-notification__content">
+                        <h5 class="p-notification__title">Information</h5>
+                        <p class="p-notification__message">{{ badges['message'] }}</p>
+                    </div>
+                </div>
+            {% else %}
+                <table aria-label="Badges earned">
+                    <thead>
                         <tr>
-                            <td>
-                                <a href="{{ badge['badge_template']['url'] }}" target="_blank">
-                                    <img src="{{ badge['image_url'] }}"
-                                         alt="{{ badge['image']['id'] }}"
-                                         height="100"
-                                         width="100" />
-                                </a>
-                            </td>
-                            <td>{{ badge["badge_template"]["name"] }}</td>
-                            <td>{{ badge["issued_at_date"] }}</td>
-                            {% if badge["state"]=="pending" %}
-                                <td>
-                                    <a href="{{ badge['accept_badge_url'] }}" target="_blank">
-                                        <button class="p-button--positive">Accept Badge</button>
-                                    </a>
-                                </td>
-                            {% else %}
+                            <th>Badge</th>
+                            <th>Exam</th>
+                            <th>Date of Issue</th>
+                            <th>Actions</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for badge in badges %}
```

Comment:
Not sure if in scope of this PR, but there doesn't seem to be any empty state. If I don't have badges it's just empty table (just able header) which looks strange.

<img width="1286" alt="image" src="https://github.com/canonical/ubuntu.com/assets/83575/44d0adc9-756d-49a2-a0e3-5c075e29e4eb">